packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -621,7 +621,7 @@ class FlutterDriver {
   }) async {
     return _sendCommand(GetDiagnosticsTree(
       finder,
-      DiagnosticsType.renderObject,
```

Comment:
note: you can see that this was exactly the same implementation as `getRenderObjectDiagnostics`:

```dart
  Future<Map<String, Object>> getRenderObjectDiagnostics(
      SerializableFinder finder, {
      int subtreeDepth = 0,
      bool includeProperties = true,
      Duration timeout,
  }) async {
    return _sendCommand(GetDiagnosticsTree(
      finder,
      DiagnosticsType.renderObject,
      subtreeDepth: subtreeDepth,
      includeProperties: includeProperties,
      timeout: timeout,
    ));
  }
```